go.mod

```diff
@@ -0,0 +1,192 @@
+module github.com/ignite/apps
```

Comment:
Why do we need a `go.mod` file at the root of repository?

Reply:
We can use the tools file; if not, don't make sense to have a makefile since we removed the `go.work`. The tools file inside the integration folder does not work anymore

Reply:
lint pipeline is still failing so we might need to change some stuff in the Makefile

Reply:
yes. The lint can't recognize the folders without the workspace, I will try to figure out